docs/templates.md

```diff
@@ -69,6 +69,8 @@ The following functions are defined.
   non-empty lines by the given `prefix`.
 * `label(label: Template, content: Template) -> Template`: Apply label to
   the content. The `label` is evaluated as a space-separated string.
+* `surround(prefix: Template, suffix: Template, content: Template) -> Template`:
+  Surround **non-empty** content with texts such as parentheses.
```

Comment:
Nit: I feel like this function is most closely related to `separate()`, so move it next to that?

Reply:
Done. I moved it there because it's single-content reformatting function, but it's also related to `separate()`.